config/provider.js

```diff
@@ -29,7 +29,7 @@ module.exports.provider = {
   grant_types_supported: ['authorization_code'],
   features: {
     devInteractions: false,
-    discovery: true,
+    discovery: false,
```

Comment:
you should leave this at "true" discovery is another part of the documentation and some FS use that path in them script to build them request.

Reply:
done. Thx !